src/app/i18n/language-switcher/switcher.js

```diff
@@ -1,13 +1,19 @@
 
 class LanguageSwitcherController {
   /** @ngInject */
-  constructor($translate) {
+  constructor($document, $translate) {
     this.$translate = $translate;
+    this.$document = $document;
   }
 
   onSwitcherClicked() {
     const currentLang = this.$translate.use();
     const newLang = currentLang === 'en' ? 'mn' : 'en';
+    if (newLang === 'mn') {
+      this.$document.find('html').addClass('cyrillic');
+    } else {
+      this.$document.find('html').removeClass('cyrillic');
```

Comment:
True, although I think I'll leave it as is since this is guaranteed to always apply the proper class.